modules/scanning-images.adoc

```diff
@@ -142,17 +107,72 @@ Scanner identifies vulnerabilities in images that contain the following Linux di
 | `sles:11`^[2]^, `sles:12`^[2]^, `sles:15`^[2]^, `opensuse-leap:15.0`^[2]^, `opensuse-leap:15.1`^[2]^
 
 | link:http://releases.ubuntu.com/[Ubuntu]
-| `ubuntu:14.04`, `ubuntu:16.04`, `ubuntu:18.04`, `ubuntu:20.04`,`ubuntu:22.04`, `ubuntu:23.10`, `ubuntu:24.04`
+| `ubuntu:14.04`, `ubuntu:16.04`, `ubuntu:18.04`, `ubuntu:20.04`, `ubuntu:22.04`, `ubuntu:24.04`, `ubuntu:24.10`
 
 The following vulnerability sources are not updated by the vendor:
-`ubuntu:12.04`, `ubuntu:12.10`, `ubuntu:13.04`, `ubuntu:14.10`, `ubuntu:15.04`, `ubuntu::15.10`, `ubuntu::16.10`, `ubuntu:17.04`, `ubuntu:17.10`, `ubuntu:18.10`, `ubuntu:19.04`, `ubuntu:19.10`, `ubuntu:20.10`, `ubuntu:21.04`, `ubuntu:21.10`, `ubuntu:22.10`, `ubuntu:23.04`, `debian:8`^[1]^, `debian:9`^[1]^, `debian:10`^[1]^
+`ubuntu:12.04`^[1]^, `ubuntu:12.10`^[1]^, `ubuntu:13.04`^[1]^, `ubuntu:14.10`^[1]^, `ubuntu:15.04`^[1]^, `ubuntu::15.10`^[1]^, `ubuntu::16.10`^[1]^, `ubuntu:17.04`^[1]^, `ubuntu:17.10`^[1]^, `ubuntu:18.10`^[1]^, `ubuntu:19.04`^[1]^, `ubuntu:19.10`^[1]^, `ubuntu:20.10`^[1]^, `ubuntu:21.04`^[1]^, `ubuntu:21.10`^[1]^, `ubuntu:22.10`^[1]^, `ubuntu:23.04`^[1]^, `ubuntu:23.10`^[1]^
 |===
 . Only supported in the StackRox Scanner.
 . Only supported in Scanner V4.
 . Images older than June 2020 are not supported in Scanner V4.
 
 [NOTE]
 ====
-* Scanner does not support the Fedora operating system because Fedora does not maintain a vulnerability database.
+Scanner does not support the Fedora operating system because Fedora does not maintain a vulnerability database.
 However, Scanner still detects language-specific vulnerabilities in Fedora-based images.
 ====
+
+[id="supported-package-formats_{context}"]
+== Supported package formats
+
+Scanner can check for vulnerabilities in images that use the following package formats:
+
+[cols="2",options="header"]
+|===
+| Package Format | Package Managers
+
+| apk
+| apk
+
+| dpkg
+| apt, dpkg
+
+| rpm
+| dnf, microdnf, rpm, yum
+|===
+
+[id="supported-programming-languages_{context}"]
+== Supported programming languages
+
+Scanner can check for vulnerabilities in dependencies for the following programming languages:
+
+[cols="2",options="header"]
+|===
+| Programming Language | Package Format
```

Comment:
```suggestion
| Programming language | Package format
```
Sorry, I just realized this needs to be in sentence-style caps per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#tables).